core/src/main/java/com/linecorp/armeria/common/RequestHeadersBuilder.java

```diff
@@ -126,6 +126,11 @@ default RequestHeadersBuilder acceptLanguages(LanguageRange... acceptedLanguages
      */
     RequestHeadersBuilder cookies(Cookie... cookies);
 
+    /**
+     * Sets the <a href="https://datatracker.ietf.org/doc/html/rfc7235#section-4.2">authorization</a> header.
+     */
+    RequestHeadersBuilder authorization(String authorization);
```

Comment:
Thanks! I know it doesn't really need to be a part of this PR, but how about adding `String RequestHeaderGetters.authorization()` as well? You're gonna have to:

- Add the getter method declarations in `RequestHeaderGetters` and `RequestHeaders`;
- Add the package-private getter implementation at `HttpHeadersBase` (no need to define a setter though); and
- Add the public getter implementation that calls `HttpHeadersBase.authorization()` at `DefaultRequestHeaders`.

If you're not sure, please take a look at how we implemented `RequestHeaders.method()` and `RequestHeaderGetters.method()` in `HttpHeadersBase` and `DefaultRequestHeaders`.

If you feel like this shouldn't be a part of this PR, you could just revert this and consider sending a follow-up PR.

Reply:
I will revert this and make a separate PR.
Thanks for giving me a solid guidance! 👍